pinot-common/src/main/java/org/apache/pinot/common/exception/QueryException.java

```diff
@@ -130,6 +131,8 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTracePerFrame) {
   public static final ProcessingException UNKNOWN_COLUMN_ERROR = new ProcessingException(UNKNOWN_COLUMN_ERROR_CODE);
   public static final ProcessingException UNKNOWN_ERROR = new ProcessingException(UNKNOWN_ERROR_CODE);
   public static final ProcessingException QUOTA_EXCEEDED_ERROR = new ProcessingException(TOO_MANY_REQUESTS_ERROR_CODE);
+  public static final ProcessingException TABLES_NOT_ON_SAME_TENANT_ERROR =
+      new ProcessingException(TABLES_NOT_ON_SAME_TENANT_CODE);
```

Comment:
i dont think we need a new code for this one. let's just use 
`BROKER_REQUEST_SEND_ERROR_CODE` 
b/c it is actually broker decided not to dispatch the query in the first place